packages/ui/Scrollable/Scrollable.js

```diff
@@ -182,6 +182,34 @@ class ScrollableBase extends Component {
 		 */
 		onMouseDown: PropTypes.func,
 
+		/**
+		 * Called when the edges are reached.
```

Comment:
How about not plural but singular for edge? It is called when one of the edges is reached.